mantidproject/mslice
# Remove MDEventWorkspace dependency on parent

Removes the search for a parent workspace before doing some operations. This was previously necessary to circumvent a bug in `SliceMD` that is now fixed. It should also be possible to load `_QE` / `_ThE` workspaces (skipping the calculate projections step) and perform all operations with them (e.g. change intensity). 

**To test:**
- Load a projected workspace directly
- Plot a slice, cut, change intensity options etc and check nothing breaks.

Fixes #226
